docs/source/roadmap.rst

```diff
@@ -60,9 +66,19 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v2.9
+
+* **Orquesta Second Beta:** Second beta of new "Orquesta" workflow engine.
+* **WebUI:** Real-time streaming output, and Inquiries support
+* **Action Output Structure Definition:** Enable optional definition of action payload, so that it
+  can be inspected and used when passing data between actions in workflows.
+* **ChatOps:** Support for Microsoft Teams.
+* **k8s:** Reference configuration for running |st2| Enterprise on k8s.
```

Comment:
Did you mean to mark this one done? It moved to the 3.0 roadmap above.

Reply:
Check the fine print. 
2.9: `* **k8s:** Reference configuration for running |st2| Enterprise on k8s.`
3.0: `* **k8s:** Reference configuration for running |st2| Community on k8s.`